src/NuGet.Core/NuGet.Configuration/PackageSourceMapping/PackageSourceMapping.cs

```diff
@@ -20,25 +22,30 @@ public class PackageSourceMapping
         /// </summary>
         internal IReadOnlyDictionary<string, IReadOnlyList<string>> Patterns { get; }
 
-        private Lazy<SearchTree> SearchTree { get; }
+        private Lazy<SearchTree?> SearchTree { get; }
```

Comment:
Would it make more sense to just have the full lazy be null. I think we can do that safely with the is enabled check.

Reply:
Then we have to null check the property itself. Not sure if I see the benefits clearly, yet...